app/Models/Photo.php

```diff
@@ -173,13 +176,8 @@ public function total ()
             if ($this->$category)
             {
                 // We dont want to include brands in total_litter
-                if ($category === 'brands')
-                {
-                    $this->$category->total();
-                }
-
                 // Increment total_litter when its not brands
-                else
+                if ($category !== 'brands')
```

Comment:
We still need to call the first block, which calculates and saves `total_brands` on the brands model.

The second block increments `total_litter`, which is a sum of all litter, expect brands

We could brands and litter separately which is why this block is needed here

Reply:
Actually $this->$category->total() does not do anything on the LitterCategory models, it doesn't increase their counter. Or am I missing something?
Anyway I'm setting the method's signature as it was before.

Reply:
Actually you are right, this line can be deleted. `total()` returns a total count of all items on that modal, which is used in some places but not here. Your original code was correct and can be accepted